lessons/pydata/pandas_core/index.ipynb

```diff
@@ -0,0 +1,3402 @@
+{
```

Comment:
<p>Series jsou tu s malým <code>s</code>.</p><p>Asi nerozumím úkolu. Mám si vybrat nějaký další sloupec z pokémonů?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p><code>series</code> má tady opravdu být malými písmeny protože to je ta konkrétní <code>Series</code> která se vytvořila předtím - ale chtělo by to přejmenovat, jinak se to zdáse bude plést :)</p>

Reply:
<p>Přejmenováno a možnost vybrání dalšího sloupce přidána.</p>